ls1intum/Artemis
# Updating Jira, Bitbucket and Bamboo version for Docker setup

### Checklist
- [x] I stepped through the guide and set up my local Bitbucket + Bamboo instance

### Motivation and Context


The independence of a specific default branch requires the support of a REST call to Bitbucket. The REST call is only supported since version 7.5.0, which requires an update since currently version 7.1.0 is specified. 

### Description

Updating the versions to those which are currently used on the production environment.
JIRA `8.13.5`
Bitbucket `7.10.1`
Bamboo `7.2.3` _(already in use)_

### Steps for Testing

If you already have a Bitbucket + Bamboo setup
1. remove the docker containers
2. rebuild the docker containers with the new versions
3. check if everything still works as expected and all versions were updated properly

If you only delete the containers and not the images you should not need to re-configure the setup. (However, this is no guarantee, as far as I experienced it the config will be stored in the images, but I am not a docker expert and might be mistaken.)
From my personal experience updating Jira in this way lead to some issues, so maybe there is a better way (I guess it was related to me using the built-in database).

In case you don't have a local Bitbucket + Bamboo setup
1. follow the [setup guide ](https://artemis-platform.readthedocs.io/en/latest/dev/setup/bamboo-bitbucket-jira/#)
2. Check whether programming exercise-related features (creation + participation) work as expected
